packages/react/src/contexts/session.context.tsx

```diff
@@ -66,6 +70,7 @@ export function SessionContextProvider({ api, data, children }: SessionContextPr
   const [isProcessing, setIsProcessing] = useState(false);
   const [storedSignature, setStoredSignature] = useState<string>();
   const [storedAddress, setStoredAddress] = useState<string>();
+  const tokenCache = useRef<Map<string, string>>(new Map());
 
```

Comment:
Do you need the ref, or can you use a normal state?

Reply:
I think in this case ref is more efficient as it avoids re-renders which shouldn't be necessary 🤔